e2e/scripts/check-installation-state.sh

```diff
@@ -51,6 +51,7 @@ main() {
     # if this is the current version in CI
     if echo "$version" | grep -qvE "(pre-minio-removal|1.8.0-k8s|previous-stable)" ; then
         validate_data_dirs
+        check_join_command
```

Comment:
feels like this should be a dryrun test instead similar to this? https://github.com/replicatedhq/embedded-cluster/blob/c74d25cc007c258532e0af3fe55e7fd60a1284d6/tests/dryrun/join_test.go#L206

Reply:
actually, nevermind. can keep it here to test the feature e2e.

Reply:
Could be both